src/components/SlateEditor/plugins/concept/inline/index.tsx

```diff
@@ -74,17 +75,30 @@ export const inlineConceptPlugin = (editor: Editor) => {
   editor.isInline = (element: Element) => {
     if (element.type === TYPE_CONCEPT_INLINE) {
       return true;
-    } else {
-      return nextIsInline(element);
     }
+    return nextIsInline(element);
   };
 
   editor.onKeyDown = (e: KeyboardEvent) => {
+    if (!editor.selection) return nextOnKeyDown?.(e);
     if (e.key === KEY_BACKSPACE) {
-      onBackspace(e, editor, nextOnKeyDown);
-    } else if (nextOnKeyDown) {
-      nextOnKeyDown(e);
+      return onBackspace(e, editor, nextOnKeyDown);
     }
+
+    if (e.key === KEY_ENTER) {
+      const [entry] = Editor.nodes<ConceptInlineElement>(editor, {
+        match: (node) => Element.isElement(node) && node.type === "concept-inline",
+        at: editor.selection,
+        mode: "lowest",
+      });
+
+      if (entry) {
+        e.preventDefault();
+        return;
+      }
+    }
```

Comment:
Trengs denne? Gjør det noe at forklaringa splittes i to slik som alle andre felt gjør?

Reply:
Det var gjort fordi du fikk opp `ConceptModalen` når du trykka `Enter`  men fant ut hvorfor nå! Så da kan vi fjerne den biten!

Reply:
No funker det som forventa. 
Men om du splitter og slår sammen en h2 så ender du opp med en h2. Om du splitter og slår sammen en lenke eller forklaring så får du to forklaringer som står intil kvarandre. Det må kanskje berre bli slik?

Reply:
Docs google gjør det samme, det må brukeren kunne håndtere.